decision-records/000-template.md

```diff
@@ -2,19 +2,36 @@
 <!-- The title should be a short present tense imperative phrase, less than 50
      characters, like a git commit message. -->
 
+## Decision
+
+<!-- An overview of the decision made. -->
+
 **Date:** <!-- YYYY-MM-DD -->
 
-**Status:** <!-- proposed, accepted, rejected, deprecated, superseded -->
+## Rationale
 
-## Context
+<!-- The reason for the decision and any comments or disagreements cited. -->
 
-<!-- What is the issue that we're seeing that is motivating this decision or
-     change. -->
+## Risks
 
-## Decision
+<!-- Any accepted risks associated with the decision. -->
+
+## Alternatives considered
+
+<!-- Other options discussed but decided against. -->
+
+## Dependencies
```

Comment:
Other decision records call these **Implications** (as they're not always dependencies). Gives us a good place to log things that we know to be aware of going forward, without it necessarily being other work

What do you think?